tez-mapreduce/src/main/java/org/apache/tez/mapreduce/hadoop/MRJobConfig.java

```diff
@@ -131,6 +131,8 @@ public interface MRJobConfig {
 
   public static final String CACHE_ARCHIVES_VISIBILITIES = "mapreduce.job.cache.archives.visibilities";
 
+  public static final String MR_JOB_UUID = "mapreduce.job.uuid";
```

Comment:
Can we javadoc on the usage of this config ? Like downstream apps like Hive, pig etc can use this to set committer id etc.

Reply:
Added javadoc description.